apps/ledger-live-mobile/src/screens/Settings/General/AnalyticsRow.tsx

```diff
@@ -16,7 +16,7 @@ const AnalyticsRow = () => {
   const { t } = useTranslation();
   const dispatch = useDispatch();
   const [isOpened, setIsOpened] = useState(false);
-  const analyticsEnabled: boolean = useSelector(analyticsEnabledSelector);
+  const analyticsEnabled: boolean = !!useSelector(analyticsEnabledSelector);
```

Comment:
Maybe it's better to use `Boolean()` instead 

https://www.tektutorialshub.com/typescript/typescript-boolean/#boolean-global-function